dev-libs/libassuan/libassuan-2.5.1.recipe

```diff
@@ -80,5 +84,6 @@ INSTALL()
 		$binDir/libassuan-config
 
 	packageEntries devel \
-		$developDir $binDir/libassuan-config
+		$developDir \
+		$binDir/libassuan-config
```

Comment:
```diff
 	packageEntries devel \
 		$developDir \
-		$binDir/libassuan-config
+		$binDir/libassuan-config \
+		$dataDir
```
(This will move `data/aclocal/libassuan.m4` to the devel package.)

Reply:
Any reason why not to include the *.info file? I see multiple files in the documentation dir?